premis3.owl

```diff
@@ -0,0 +1,733 @@
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/copyrightStatus"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Dependency">
+    <rdfs:label>Dependency</rdfs:label>
+    <rdfs:comment>Relationship where one Object requires another Object to support its function, delivery, or
+      the coherence of its content.</rdfs:comment>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Environment">
+    <rdfs:label>Environment</rdfs:label>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/Object"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Event">
+    <rdfs:label>Event</rdfs:label>
+    <rdfs:comment>Preservation action that involves at least one digital object and/or Agent.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.w3.org/ns/prov#Activity"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/File">
+    <rdfs:label>File</rdfs:label>
+    <rdfs:comment>Named and ordered sequence of bytes that is known to an operating system.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/Object"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Fixity">
+    <rdfs:label>Fixity</rdfs:label>
+    <rdfs:comment>Information used to verify whether an object has been altered in an undocumented or
+      unauthorized way.</rdfs:comment>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Format">
+    <rdfs:label>Format</rdfs:label>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/HardwareAgent">
+    <rdfs:label>Hardware Agent</rdfs:label>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Identifier">
+    <rdfs:label>Identifier</rdfs:label>
+    <rdfs:comment>An unambiguous reference to the PREMIS entity within the preservation
+      repository.</rdfs:comment>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/IntellectualEntity">
+    <rdfs:label>Intellectual Entity</rdfs:label>
```

Comment:
Shouldn't we declare subclasses of Object (and probably also RightsBasis and Agent) disjoint classes?

Reply:
I've added `owl:disjointWith` statements declaring the subclasses of `premis:Object` disjoint, and done the same for `premis:Agent`.